### ISSUE TITLE
Consider toml.Decode vs toml.Unmarshal

### ISSUE THREAD

**Issue by ccoVeille** [butuzov/mirror#76]
I thought about your project by reading this PR

- https://github.com/mgechev/revive/pull/1122

https://pkg.go.dev/github.com/BurntSushi/toml#Unmarshal

https://pkg.go.dev/github.com/BurntSushi/toml#Decode

https://github.com/BurntSushi/toml/blob/v1.4.0/decode.go#L24-L37

Could you have interest for supporting it?

**Comment by butuzov** [comment]
Yeah, it looks its "our" case and @golangci is already have its supported. I will gladly except PR from you or @alexandear if you are going to create one. I will also try to find time once ... once i have chance to sit calmly for an hour.

Cheers!

**Comment by ccoVeille** [comment]
> and golangci is already have its supported

I might have missed something then.

What do you mean?

**Comment by alexandear** [comment]
As I understand it, the linter `mirror` is designed to only support finding mirroring functions in the standard library, not in third-party libraries like `github.com/BurntSushi/toml`.

**Comment by butuzov** [comment]
Correct! 
1) std lib first. However this package supported by main engine (i am considering `gonagci-lint` main engine for running `mirror`), so it can be OK to have its support. 
2) Issue I see now - `Unmarshal` and `Decode` has different signatures, therefore its not really really suitable for mirror (as alt function will need to have almost same signature).
